docs/content/en/docs/reference/clusterspec/nutanix.md

```diff
@@ -0,0 +1,322 @@
+---
+title: "Nutanix configuration"
+linkTitle: "Nutanix configuration"
+weight: 20
+description: >
+  Full EKS Anywhere configuration reference for a Nutanix cluster.
+---
+
+This is a generic template with detailed descriptions below for reference.
+The following additional optional configuration can also be included:
+
+* [CNI]({{< relref "optional/cni.md" >}})
+* [IAM for pods]({{< relref "optional/irsa.md" >}})
+* [IAM Authenticator]({{< relref "optional/iamauth.md" >}})
+* [OIDC]({{< relref "optional/oidc.md" >}})
+* [gitops]({{< relref "optional/gitops.md" >}})
+* [proxy]({{< relref "optional/proxy.md" >}})
+* [Registry Mirror]({{< relref "optional/registrymirror.md" >}})
```

Comment:
none of these besides maybe the CNI options are supported yet so I don't think we should mention them here